tests/test_util.py

```
@@ -658,5 +658,645 @@ def test_consistency_across_languages(self):
+            result = _normalize_number_word("123", "en", mock_engine)
+            self.assertEqual(result, "123")
+
+    @patch('phoonnx.util.is_numeric')
+    def test_normalize_number_word_european_format_edge_cases(self, mock_is_numeric):
+        """Test _normalize_number_word with European format edge cases."""
+        mock_is_numeric.side_effect = lambda x: x == "1000000.0"
+        
+        with patch('phoonnx.util.pronounce_number') as mock_pronounce:
+            mock_pronounce.return_value = "one million"
+            # Test large number with European decimal separator
+            result = _normalize_number_word("1.000.000,0", "pt", None)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)